static/app/utils/useDevicePixelRatio.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+/**
+ * Hook to retrieve dpr value of the device and monitor for changes
+ * (e.g. if user drags window to a screen with different dpr, we want to be notified).
+ * @returns dpr of the device
+ */
+function useDevicePixelRatio(): number {
+  const [devicePixelRatio, setDevicePixelRatio] = React.useState<number>(
+    window.devicePixelRatio
+  );
```

Comment:
You could simplify this with
```
const [devicePixelRatio, updateDevicePixelRatio] = React.useReducer(
  () => window.devicePixelRatio,
  window.devicePixelRatio
);
```

Reply:
Hmm, why would you use a useReducer instead of useState here? The general convention for setState naming is to have [state setState] and not [state, updateState]